src/components/AnonymousChat/BaseChatItem.tsx

```diff
@@ -82,21 +79,24 @@ const BaseChatItem = ({item, index, type}: BaseChatItemComponentProps) => {
         message={item?.message}
         time={calculateTime(item?.updatedAt, true)}
         username={handleUserName(item)}
-        type={BaseChatItemTypeProps.MY_ANON_CHAT}
         status={item?.status as ChatStatus}
-        chatType={type}
+        chatType={type ?? ANONYMOUS}
+        messageType={item?.type}
+        data={item?.rawJson}
       />
     );
 
   return (
     <ChatItemTargetText
       key={index}
       avatar={handleAvatar()}
-      isContinuous={item?.isContinuous}
+      isContinuous={item?.isContinuous ?? false}
       message={item?.message}
       time={calculateTime(item?.updatedAt, true)}
       username={handleUserName(item)}
-      type={BaseChatItemTypeProps.ANON_CHAT}
+      chatType={type ?? ANONYMOUS}
+      messageType={item?.type ?? 'regular'}
+      data={item?.rawJson}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes here seem to be primarily about providing default values for certain properties (`chatType`, `messageType`, `isContinuous`). This is generally a good practice as it can prevent potential runtime errors due to undefined values. However, please ensure that these default values are appropriate and won't introduce unexpected behavior.
```diff
-        type={BaseChatItemTypeProps.MY_ANON_CHAT}
         status={item?.status as ChatStatus}
-        chatType={type}
+        chatType={type ?? ANONYMOUS}
+        messageType={item?.type}
+        data={item?.rawJson}

...

-      type={BaseChatItemTypeProps.ANON_CHAT}
+      isContinuous={item?.isContinuous ?? false}
+      messageType={item?.type ?? 'regular'}
+      data={item?.rawJson}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->